frontend/src/Header.js

```diff
@@ -24,6 +26,15 @@ function Header() {
                     </Nav.Link>}
                     {checkPermission && user.role === "ROLE_SYSTEM_ADMIN" &&
                     <Nav.Link className={customersClass} href="/customers">Customers</Nav.Link>}
+                    {checkPermission && (user.role === "ROLE_SYSTEM_ADMIN" || user.role === "ROLE_DISPATCHER" ||
```

Comment:
checkPermission && (user.role === "ROLE_SYSTEM_ADMIN" || user.role === "ROLE_DISPATCHER" ||
                        user.role === "ROLE_LOGISTICS_SPECIALIST" || user.role === "ROLE_ADMIN" can be extracted in a separate variable

Reply:
done